server/routes/index.js

```diff
@@ -0,0 +1,24 @@
+import client from '../helpers/redis';
+
+import daysBetween from '../helpers/daysBetween';
+
+export default (app) => {
+  app.get('/worker-status', (req, res) => client.mget('startTime', 'message', 'jobDone', (err, response) => {
+    if (err) {
+      return res.status(400).send({
+        message: 'Error with redis connection, kindly connect to redis',
+      });
+    }
+
+    const startTime = new Date(JSON.parse(response[0]));
+    const message = response[1];
```

Comment:
You can use an array destructuring here like:
`const [message, numberOfJobs] = response`